.storybook/CmsTheme.js

```diff
@@ -0,0 +1,36 @@
+import { create } from '@storybook/theming';
```

Comment:
We can update this and "brand" it more. I used a few colors from core.
Branding will be more important if we host storybook somewhere